csharp/BUILD.bazel

```diff
@@ -51,3 +52,9 @@ pkg_files(
     strip_prefix = strip_prefix.from_root(""),
     visibility = ["//pkg:__pkg__"],
 )
+
+sh_binary(
+  name = "release",
+  srcs = ["build_release.sh"],
+  args = ["$(location build_release.sh)"],
```

Comment:
what is this for? I don't see it used in the script

Reply:
This is the bazel target to run the bash script so we can just `bazel run //csharp:release`